pkg/utils/fileHandling.go

```diff
@@ -0,0 +1,56 @@
+package utils
+
+import (
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// EditFile will edit the content of a file
+func EditFile(filepath string, old string, new string) error {
+	input, err := ioutil.ReadFile(filepath)
+	if err != nil {
+		return errors.Wrapf(err, "Failed to read the given file, due to:%v", err)
```

Comment:
Yes, because the format of Wrapf is:
`func Wrapf(err error, format string, args ...interface{}) error`
https://godoc.org/github.com/pkg/errors#example-Wrapf
And the formate of Wrap is:
```
_, err := ioutil.ReadAll(r)
if err != nil {
        return errors.Wrap(err, "read failed")
}
```

Reply:
ref: https://github.com/litmuschaos/chaos-runner/blob/master/pkg/utils/expNotFoundPatch.go#L13